connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsIntegrationBaseTest.java

```diff
@@ -455,27 +470,44 @@ private static void waitUntilMirrorMakerIsRunning(EmbeddedConnectCluster connect
                     "Connector " + connector.getSimpleName() + " tasks did not start in time on cluster: " + connectCluster);
         }
     }
- 
+
+    /*
+     * wait for the topic created on the cluster
+     */
+    private static void waitForTopicCreated(EmbeddedKafkaCluster cluster, String topicName) throws InterruptedException {
+        try (final Admin adminClient = cluster.createAdminClient()) {
+            waitForCondition(() -> adminClient.listTopics().names().get().contains(topicName), OFFSET_SYNC_DURATION_MS,
```

Comment:
Why are we using `OFFSET_SYNC_DURATION_MS`? This does not seem related to offset sync. Maybe we can just use a literal value. WDYT?

Reply:
You're right, I created a `TOPIC_SYNC_DURATION_MS` for it.